src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -82,7 +82,7 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
 
     @Input
     @Optional
-    final MapProperty<String, String> labels = project.objects.mapProperty(String, String)
+    MapProperty<String, String> labels = project.objects.mapProperty(String, String)
```

Comment:
Can you please add a comment via Javadoc that documents the design decision? While we are at it, please also explain `labels` represents as part of the comment.

Reply:
Sure, Here is another PR to add documentation for all properties. Once it is merged, I will rebase the current MR on top of it and address you comment

Reply:
Merged it yesterday.

Reply:
Done